charts/datadog/templates/_container-security-agent.yaml

```diff
@@ -7,7 +7,12 @@
     capabilities:
       add: ["AUDIT_CONTROL", "AUDIT_READ"]
   {{- end }}
-  command: ["security-agent", "start", "-c={{ template "datadog.confPath" . }}/datadog.yaml"]
+  {{- if eq .Values.targetSystem "linux" }}
+  command: ["env", "DD_LOG_LEVEL={{ .Values.agents.containers.securityAgent.logLevel | default .Values.datadog.logLevel }}", "security-agent", "start", "-c={{ template "datadog.confPath" . }}/datadog.yaml"]
```

Comment:
I see some containers have a `| quote` filter on the log level (`otel-agent` ) and others don't. Why the difference?